src/test-run/browser-dialogs.js

```diff
@@ -0,0 +1,25 @@
+// -------------------------------------------------------------
+// WARNING: this file is used by both the client and the server.
+// Do not use any browser or node-specific API!
+// -------------------------------------------------------------
+export const TYPE = {
+    beforeUnload: 'beforeunload',
+    alert:        'alert',
+    confirm:      'confirm',
+    prompt:       'prompt'
+};
+
+export class ExpectedDialog {
+    constructor (type, result) {
+        this.type   = type;
+        this.result = result;
```

Comment:
We also have `result` here https://github.com/DevExpress/testcafe/pull/621/files#diff-e2bf5c5891285c6f72a00b881eb9b4a5R235
may be just `returnValue`?